sys/net/sock/sock_util.c

```diff
@@ -116,30 +117,35 @@ static char* _find_pathstart(const char *url)
 
 int sock_urlsplit(const char *url, char *hostport, char *urlpath)
 {
+    assert(url);
```

Comment:
```suggestion
    assert(url != NULL);
```

Reply:
please keep the original authors style where it doesn't go against coding conventions.